circuits/cpp/src/aztec3/circuits/rollup/test_utils/utils.cpp

```diff
@@ -1,10 +1,18 @@
+#include "aztec3/circuits/abis/membership_witness.hpp"
+#include "aztec3/circuits/rollup/base/init.hpp"
+#include "barretenberg/numeric/uint256/uint256.hpp"
+#include "barretenberg/stdlib/merkle_tree/memory_store.hpp"
+#include "barretenberg/stdlib/merkle_tree/merkle_tree.hpp"
 #include "nullifier_tree_testing_harness.hpp"
 #include "utils.hpp"
 #include "aztec3/constants.hpp"
 #include "init.hpp"
 
 #include <aztec3/circuits/kernel/private/utils.hpp>
 #include <aztec3/circuits/mock/mock_kernel_circuit.hpp>
+#include <cstddef>
```

Comment:
I dont think these are required?

Reply:
Removed! Not sure how they got there, maybe vs code was trying to be too smart...?

Reply:
they always creep into my files  too!